auth0/reset-tenant.bash

```
@@ -0,0 +1,59 @@
+# Delete writer application
+if [ ! -z $(curl -s -H "Authorization: Bearer $AT" https://$TENANT/api/v2/clients?fields=name | jq -r '.[] | select(.name == "writer") | .name') ]
+then
+  echo "Delete writer client app..."
+  writerId=$(curl -s -H "Authorization: Bearer $AT" https://$TENANT/api/v2/clients | jq -r '.[] | select(.name == "writer") | .client_id')
+  curl -s -H "Authorization: Bearer $AT" -X DELETE https://$TENANT/api/v2/clients/$writerId
+else
+  echo "Writer client app already deleted"
+fi
+
+# Delete the API
+if [ ! -z $(curl -s -H "Authorization: Bearer $AT" "https://$TENANT/api/v2/resource-servers" | jq -r ".[] | select(.name == \"$API_NAME\") | .name") ]
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)